packages/react/src/components/PageHeader/PageHeader-test.js

```diff
@@ -16,7 +16,7 @@ import {
 } from '../PageHeader';
 import * as hooks from '../../internal/useMatchMedia';
 import { breakpoints } from '@carbon/layout';
-
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from '../Tabs/Tabs';
```

Comment:
I don't think we need `Tabs` since we're using `PageHeader.Tabs` instead

```suggestion
import { TabList, Tab, TabPanels, TabPanel } from '../Tabs/Tabs';
```